en/components/stepper.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Stepper - Design System Component
+_description: The Stepper Component Symbol conveys progress through numbered steps.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits, Figma, Figma to Angular, Export code from Figma, Figma HTML, Figma to HTML, Figma UI kits
```

Comment:
Consider removing the Sketch keywords because we are not supporting the Stepper in Sketch

Reply:
@sdimchevski should I do this? We were also wondering whether I should add notes on the new component pages, that currently we don't support the component in Sketch and Adobe XD.

Reply:
We can say the Stepper is coming to Sketch in the near future and leave out the Sketch keywords for now

Reply:
Fixed